MonstieWash/Assets/Scripts/Player/PlayerHealth.cs

```diff
@@ -163,5 +185,52 @@ IEnumerator StartInvincibility()
         }
 
         m_isInvincible = false;
-    }    
+    }
+
+    /// <summary>
+    /// Applies a slow and knockback away from the centre of the monster, knockback and slow reduce over time.
+    /// </summary>
+    /// <returns></returns>
+    IEnumerator ApplyKnockbackEffects()
+    {
+        //Initialise time tracking and set hand move speed to starting slow strength.
+        var timeRunning = 0f;
+        m_hand.HandSpeed = Mathf.Max(m_originalMoveSpeed - speedCurve.Evaluate(0 / slowDuration) * slowStrength, 0);
```

Comment:
Not sure if this is an oversight but dividing 0 by anything will always give 0. I assume it's meant to be `timeRunning / slowDuration`?

```suggestion
        //Initialise time tracking and set hand move speed to starting slow strength.
        var timeRunning = 0f;
        m_hand.HandSpeed = Mathf.Max(m_originalMoveSpeed - speedCurve.Evaluate(timeRunning / slowDuration) * slowStrength, 0);
```